src/reporting/feedback.tsx

```diff
@@ -74,17 +76,21 @@ const TAContainer = styled(Container)`
 	}
 `;
 
-const SubHeadingText = styled(Text)`
+const SubHeadingText = styled(Text)<{ lineHeight?: string }>`
 	border-radius: 2px 2px 0 0;
 	line-height: 21px;
 	font-size: 14px;
 	font-weight: 300;
 	margin-top: 10px;
-	line-height: ${(props): string => props.lineHeight};
+	line-height: ${({ lineHeight }): string => lineHeight ?? 'normal'};
 `;
 
-const LabelContainer = styled(Container)`
-	border-bottom: 1px solid ${(props): string => (props.disabled ? 'red' : '#cfd5dc')};
+interface LabelContainerProps extends ContainerProps {
+	disabled: boolean;
+}
+
+const LabelContainer = styled(Container)<LabelContainerProps>`
+	border-bottom: 1px solid ${({ disabled }): string => (disabled ? 'red' : '#cfd5dc')};
```

Comment:
You can use theme to get the proper color from design System instead of hardcode values